clar2wasm/tests/lib_tests.rs

```diff
@@ -1148,15 +1148,15 @@ test_contract_call_response!(
 // chain, which is not the case when running the tests. Once the test framework
 // supports this, these tests can be re-enabled.
 
-test_contract_call_response!(
-    test_gbi_non_existent,
-    "get-block-info",
-    "non-existent",
-    |response: ResponseData| {
-        assert!(response.committed);
-        assert_eq!(*response.data, Value::none());
-    }
-);
+// test_contract_call_response!(
```

Comment:
remove code instead of commenting out

Reply:
@krl It's already par of a big block a commented tests.
Are you sure it can't remain like that?

Reply:
@krl Is this better https://github.com/stacks-network/clarity-wasm/pull/420/commits/457f7149e327332196a7c5a8f360b8d0e9e8f987 ?

Reply:
I'm still not sure what's happening but in general if these tests pass they should be there and not commeted. so yes

Reply:
This is the fix that makes them pass: 
https://github.com/stacks-network/clarity-wasm/commit/457f7149e327332196a7c5a8f360b8d0e9e8f987#diff-88585f598b79cb14968cf2e11093aacaa49eb5f7553ded728347b3ffafc8c410R73
(L73 and L89)

So this is now the only change in clar2wasm/tests/lib_tests.rs.